Makefile

```
@@ -29,7 +29,7 @@
 export BUILD_WITH_CONTAINER ?= 0
 
 ifeq ($(BUILD_WITH_CONTAINER),1)
-IMG = gcr.io/istio-testing/build-tools:2019-08-25T20-37-02
+IMG = gcr.io/istio-testing/build-tools:2019-08-28T11-32-19
```

This image doesn't exist.
```suggestion
IMG = gcr.io/istio-testing/build-tools:2019-08-29T13-57-48
```